examples/platform/silabs/efr32/rs911x/hal/rsi_hal_mcu_ioports.c

```diff
@@ -61,9 +60,20 @@ void rsi_hal_config_gpio(uint8_t gpio_number, uint8_t mode, uint8_t value)
 
     CMU_ClockEnable(cmuClock_GPIO, true);
 
-    // WFX_RSI_LOG ("RSI: CFG GPIO: 0x%x", gpio_number);
     switch (gpio_number)
     {
+    case RSI_HAL_SLEEP_CONFIRM_PIN:
+    case RSI_HAL_LP_SLEEP_CONFIRM_PIN:
+        GPIO_PinModeSet(WFX_SLEEP_CONFIRM_PIN.port, WFX_SLEEP_CONFIRM_PIN.pin, gpioModeWiredOrPullDown, PINOUT_SET);
+        break;
+    case RSI_HAL_WAKEUP_INDICATION_PIN:
+#ifndef LOGGING_STATS
+        GPIO_PinModeSet(WAKE_INDICATOR_PIN.port, WAKE_INDICATOR_PIN.pin, gpioModeWiredOrPullDown, PINOUT_CLEAR);
+        break;
+#else
+        GPIO_PinModeSet(LOGGING_WAKE_INDICATOR_PIN.port, LOGGING_WAKE_INDICATOR_PIN.pin, gpioModeWiredOrPullDown, PINOUT_CLEAR);
+        break;
+#endif
```

Comment:
break statements could outside the #ifndef. Same comment in all switch statements.

Reply:
Okay